db/schema.rb

```diff
@@ -43,10 +44,16 @@
     t.text "notes"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.index ["aircraft_id"], name: "index_logbook_entries_on_aircraft_id"
     t.index ["pilot_in_command_id"], name: "index_logbook_entries_on_pilot_in_command_id"
     t.index ["second_in_command_id"], name: "index_logbook_entries_on_second_in_command_id"
   end
 
+  create_table "pilots", force: :cascade do |t|
```

Comment:
run rails migrate and push the changes for the schema. pilots should not be here.